.travis.yml

```diff
@@ -12,8 +12,11 @@ git:
 script:
   - docker build . -t renku/renku-demo
 
-after_success:
-  - if [[ "$TRAVIS_BRANCH" == "master" ]]; then
-      docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD ;
-      docker push renku/renku-demo ;
-    fi
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script:
+      - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD ;
+      - docker push renku/renku-demo ;
+    on:
+      branch: travis
```

Comment:
note: it should be `master` before merging the PR

Reply:
thanks - I have that on my radar. Is there another way to test this step before merging to master?